dashboard/db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2023_05_24_221710) do
+ActiveRecord::Schema.define(version: 2023_05_30_215538) do
```

Comment:
Hmmm, this schema change shouldn't be here. Something must have happened during rebase against staging.